fix(admin): log admin server exceptions and close admin socket (#5045)
PostgREST/postgrest: CHANGELOG.md

- Fix unexpected results when embedding and filtering the same table more than once by @laurenceisla in #4075
- If the schema cache fails to reload, PostgREST will no longer stop serving requests and will continue doing so in a "best effort" basis by @mkleczek in #4873 #4869
- Stop reporting 503s errors unnecessarily while the schema cache is loading at startup by @mkleczek in #4880

> - Fix admin server dies silently and requests hang indefinitely by @taimoorzaeem in #5012

Reviewer: Is this true? So when it dies now the requests won't hang?

If not, perhaps the right fix is to also close the socket?

Reviewer: Alternatively, just put:

```suggestion
- Fix admin server not logging failure cause by @taimoorzaeem in #5012
```

Author: > Is this true? So when it dies now the requests won't hang?

Yes.
